src/Containers/Loading/Loading.tsx

```diff
@@ -15,12 +15,15 @@ export interface LoadingProps
         MainInterface {
     loading?: boolean;
     message?: string;
+    /** if true, the component is not positioned absolute */
+    isNotPositionedAbsolute?: boolean;
 }
 
 export const Loading: React.FC<LoadingProps> = ({
     children,
     loading = false,
     message = 'Loading...',
+    isNotPositionedAbsolute = false,
```

Comment:
Rename to istopOfPageLoader of something more clear that you are overriding isPageLevelLoading